gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -136,6 +136,17 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
+    parser.add_argument(
+        '--sharding_config_path',
+        default=('gcp_variant_transforms/data/sharding_configs/'
+                 'homo_sapiens_default.yaml'),
+        help=('File containing list of output tables, their name suffixes, and '
+              'approximate number of total base pairs which is used to conduct '
+              'BigQuery integer range partitioning. Default value is set to a '
+              'file which is optimized for the human genome. It results in one '
+              'table per chromosome (overall 25 BigQuery tables). For more '
+              'information visit gcp_variant_transforms/data/sharding_configs/'
+              'homo_sapiens_default.yaml'))
```

Comment:
Hmm, I actually meant to a proper doc - gcp-variant-trannsforms/docs/sharding.md. It's outdated right now, but we are going to modify it at some point to reflect the new schema, I presume.

Reply:
Sorry, done.